### PR TITLE
[NALM-202] Document how to connect to local studio

### PR DESCRIPTION
**Pull request by jogboms**

### CODE REVIEW

File: content/howto/mobile/native-builder.md [hunk 1]
```diff
@@ -198,37 +198,45 @@ If you have custom native dependencies or code, you can include them in your app
...
 For an iOS app, do the following:
 
-1. Clone your repository locally from Github
-2. Open the `ios` directory with **Xcode**
-3. Where `LOCAL_IP_ADDRESS` should be replaced by your local IP address. An example could be `10.0.0.2`.
-4. Locate the `AppDelegate.swift` file
-5. Go to line **13** and replace this section of the code;
+1. Clone your repository locally from GitHub.
+2. Open the `ios` directory using **Xcode**.
+3. Replace `LOCAL_IP_ADDRESS` with your local IP address (*10.0.0.2* for example).
```
Comment (jogboms) [inline content/howto/mobile/native-builder.md]: Maybe `replace any instance of`?

Comment (ConnorLand) [reply content/howto/mobile/native-builder.md]: Done!

File: content/howto/mobile/native-builder.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ The Native Builder takes your Mendix project containing a native profile and pac
 
 ## 3 About the Native Builder
 
-Native Builder uses MxBuild, GitHub, and App Center to build your apps. The tool automates the configuration of these processes to streamline your app building experience. When run, the Native Builder packages your apps by doing the following:
+The Native Builder uses MxBuild, GitHub, and App Center to build your apps. The tool automates the configuration of these processes to streamline your app building experience. So long as your app names are unique, you can even create multiple apps. When run, the Native Builder packages your apps by doing the following:
```
Comment (ConnorLand) [inline content/howto/mobile/native-builder.md]: Get this new addition (re: slack convo with Danny) checked by Jeremiah.

Comment (jogboms) [reply content/howto/mobile/native-builder.md]: LGTM

Comment (ConnorLand) [inline content/howto/mobile/native-builder.md]: @jogboms In this new sentence, I tried to advertise the capability you told Bart about over slack yesterday: "With the latest version of NativeBuilder ie 1.0.0, you can now create multiple apps as long as the app name stays unique."

Is my sentence here good enough, or should we elaborate more?

Comment (jogboms) [reply content/howto/mobile/native-builder.md]: Maybe; Native builder allows you to create as many applications on github as possible as long as they are given unique application names.

File: content/howto/mobile/native-builder.md [hunk 3]
```diff
@@ -200,9 +200,9 @@ In App Center you can configure your builds at the branch level. If no configura
 
 ### 8.4 Connecting to a Local Running Instance of Studio Pro
 
-If you have added custom dependencies and still need to test against a local running app in Studio Pro, you need to make some local changes to your template.
+Advanced users might wish to connect to a local running instance of Studio Pro. Be aware that if you make changes to your template, you must revert all those changes before connecting your template to the Native Builder again.
```
Comment (jogboms) [inline content/howto/mobile/native-builder.md]: Maybe: "...Be aware that if you make any changes to this effect, it detaches your application from the Native Builder. If you wish to continue using Native Builder's flow, you must revert all the following changes."

File: content/howto/mobile/native-builder.md [hunk 4]
```diff
@@ -200,9 +200,9 @@ In App Center you can configure your builds at the branch level. If no configura
...
 
-> Replace any instance of `LOCAL_IP_ADDRESS` with your local IP address ({10.0.0.2} for example).
+Note that if you have added custom dependencies and still need to test against a local running app in Studio Pro, you must replace any instance of `LOCAL_IP_ADDRESS` with your local IP address ({10.0.0.2} for example) in your template.
```
Comment (jogboms) [inline content/howto/mobile/native-builder.md]: Technically, irregardless of whether you add custom dependencies, as you read and implement this section, remember to replace any instance of `LOCAL_IP_ADDRESS ` with your local IP address.

File: content/howto/mobile/native-builder.md (hunk 3 above)
Comment (jogboms) [reply content/howto/mobile/native-builder.md]: So, they know the only changes they need to revert are the ones contained in this section only. And it doesn't affect if they add custom dependencies.

Comment (ConnorLand) [reply content/howto/mobile/native-builder.md]: Changed. Do latest changes solve this?

File: content/howto/mobile/native-builder.md (hunk 4 above)
Comment (ConnorLand) [reply content/howto/mobile/native-builder.md]: Changed. Does latest change solve this?

Comment (jogboms) [reply content/howto/mobile/native-builder.md]: I would say: "... if you make *these* changes...". So as to be more explicit

**Pull request merged**